### PR TITLE
Move Users Online backend to Web API

### PR DESCRIPTION
**Pull request by johnhenley**
### Description of PR...
Move Users Online backend from forumhandler.ashx to Web API
 

## PR Template Checklist

- [ ] Fixes Bug
- [ ] Feature solution
- [X] Other

Related to epic issue #53

### CODE REVIEW

File: Services/Controllers/UserController.cs [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    /// <inheritdoc/>
+    /// </summary>
+    public class UserController : ControllerBase<UserController>
+    {
+        /// <summary>
+        /// Fired by UI while user is online to update user's profile with 
+        /// </summary>
+        /// <returns></returns>
+        /// <remarks>https://dnndev.me/API/ActiveForums/User/UpdateUserIsOnline</remarks>
+        [HttpGet]
```
Comment (WillStrohl) [inline Services/Controllers/UserController.cs:44]: Since this is an editing action, shouldn't we be using a `POST` instead? 🤔

**Review by WillStrohl — COMMENTED** [review]
Looks good, pending a question I posed in the code review.

**Comment by WillStrohl** [comment]
@johnhenley There is a minor conflict in the `csproj` as well.

File: Services/Controllers/UserController.cs (hunk 1 above)
Comment (johnhenley) [reply Services/Controllers/UserController.cs:44]: Technically probably yes. The services framework handles passing the UserInfo object back to the API, so there is no data passed back. So I just made it a GET. I can change to a POST if you think that's more appropriate. Let me know...

Comment (johnhenley) [reply Services/Controllers/UserController.cs]: I went ahead and changed it to a `POST`.

**Comment by johnhenley** [comment]
> @johnhenley There is a minor conflict in the `csproj` as well.

DONE.

**Review by WillStrohl — APPROVED** [review]
Looks good!

**Pull request merged**